app/services/tagline_generator.rb

```diff
@@ -1,5 +1,5 @@
 class TaglineGenerator
-  def initialize(source, options={})
+  def initialize(source, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.